gpt_index/readers/chroma.py

```diff
@@ -0,0 +1,71 @@
+"""Chroma Reader."""
```

Comment:
just a heads up, we're adding readers to llamahub now but i can take care of that migration, https://github.com/emptycrown/llama-hub

Reply:
thanks!